packages/amplication-design-system/src/components/Button/Button.scss

```diff
@@ -186,7 +186,7 @@ $split-icon-font-size: 20px;
     background-color: transparent;
     color: var(--black100);
     border: 1px solid var(--black100);
-    border-radius: 20px !important; //overwrite the amp-button radius
+    border-radius: 4px;
```

Comment:
I am just wondering if there was a need for !important before, maybe it is still needed? also, the change from 20 to 4 seems extreme... did we check the impact on all places in the UI?